src/helpers/zones.ts

```diff
@@ -476,8 +476,8 @@ export function isOneDimensional(zone: Zone): boolean {
  */
 export function positions(zone: Zone): Position[] {
   const positions: Position[] = [];
-  const [left, right] = [zone.right, zone.left].sort((a, b) => a - b);
-  const [top, bottom] = [zone.top, zone.bottom].sort((a, b) => a - b);
+  const [left, right] = zone.left <= zone.right ? [zone.left, zone.right] : [zone.right, zone.left];
+  const [top, bottom] = zone.top <= zone.bottom ? [zone.top, zone.bottom] : [zone.bottom, zone.top];
```

Comment:
We could even do something  better to avoid creating new arrays:
A helper function `orderZone` or something like that. That would reorder the zone, or return the original zone object if it's already ordered (which should be most of the time)
```suggestion
  const { left, right, top, bottom } = orderZone(zone);
```